### ISSUE TITLE
Create good ChatGPT prompt

### ISSUE THREAD

**Issue by thogebati** [amosproj/amos2023ws01-ticket-chat-ai#64]
Try to create a good ChatGPT prompt to create test data in our format, so that the results are good

## Acceptance criteria
- [x] make sure data is in our format
- [x] make sure they are not to similar 

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
